src/Proto3/Suite/Types.hs

```diff
@@ -98,6 +99,13 @@ newtype Nested a = Nested { nested :: Maybe a }
   deriving (Show, Eq, Ord, Generic, NFData, Monoid, Arbitrary, Functor, Foldable,
             Traversable, Applicative, Alternative, Monad)
 
+-- | 'AlwaysEmit' provides a way to force emission of field values, even when
```

Comment:
`AlwaysEmit` is a fine name to me but curious why you didn't go with `ForceEmit`?

Reply:
Hmm, that's probably better, thanks.